packages/node/src/node/client/ClientGroupInteraction.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2022-2025 Matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { ActionContext } from "#behavior/index.js";
+import { ImplementationError } from "#general";
+import {
+    ClientInvoke,
+    DecodedInvokeResult,
+    Read,
+    ReadResult,
+    Subscribe,
+    SubscribeResult,
+    Write,
+    WriteResult,
+} from "#protocol";
+import { ClientNodeInteraction } from "./ClientNodeInteraction.js";
+
+export class ClientGroupInteraction extends ClientNodeInteraction {
+    /** Groups do not support reading or subscribing to attributes */
+    override read(_request: Read, _context?: ActionContext): ReadResult {
+        throw new ImplementationError("Groups do not support reading attributes");
```

Comment:
I think a custom `InvalidGroupOperationError extends ImplementationError` or something is warranted